{
    "docs": "https://docs.ansible.com/ansible-core/2.14/dev_guide/testing/sanity/import.html", 
    "results": [
        {
            "message": "The test `ansible-test sanity --test import --python 3.5` [[explain](https://docs.ansible.com/ansible-core/2.14/dev_guide/testing/sanity/import.html)] failed with 3 errors:", 
            "output": "plugins/module_utils/librenms_api_client.py:39:49: traceback: SyntaxError: invalid syntax\nplugins/modules/libre_devices.py:240:67: traceback: SyntaxError: invalid syntax\nplugins/modules/libre_devices_info.py:143:101: traceback: SyntaxError: invalid syntax"
        }
    ], 
    "verified": false
}
